pages/[caseStudySlug].js

```diff
@@ -77,8 +77,7 @@ export const getStaticProps = async (ctx) => {
   return {
     props: {
       model
-    },
-    revalidate: 300,
+    }
```

Comment:
This should be the same for case study pages, as well as the index page, correct?